### PR TITLE
[IT-2334] Decommission org-sagebase-sandbox account

### PR DESCRIPTION
**Pull request by zaro0508**
Remove cloudformation templates for our AWS sandbox account.
Keep it in organizations so that CI can still access and do this task.
Once complete there will be another PR to remove from organizations
which will complete the account removal.

### CODE REVIEW

**Comment by zaro0508** [comment]
Hold off until we get go ahead in issue IT-2334

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -29,14 +29,6 @@ Parameters:
     Type: String
     Default: '906769aa66-5d23a723-54f3-4c08-a67b-311e555f4e85'
 
-  sandboxDeveloperGroup:  #JC aws-sandbox-developers
-    Type: String
-    Default: '906769aa66-c66c6e77-3904-4bfc-b96d-98315a9bb913'
-
-  sandboxAdminGroup:    #JC aws-sandbox-admins
-    Type: String
-    Default: '906769aa66-0133decd-a118-4b53-b5a1-7ca6a3077212'
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:38]: need to remove from JC

File: org-formation/710-tgw/_tasks.yaml [hunk 2]
```diff
@@ -77,12 +76,6 @@ TransitGatewayRoutes:
       IncludeMasterAccount: false
   TemplatingContext:
     TgwSpokes:
-      # org-sagebase-itsandbox
-      dustbunnyvpc:
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply org-formation/710-tgw/_tasks.yaml:81]: oops. that was a mistake

[the remark this answers is not in the document]
Comment (zaro0508) [reply org-formation/710-tgw/_tasks.yaml:81]: it's been restored. thanks!

**Review by jesusaurus — APPROVED** [review]

**Pull request merged**